test/integration-tests/IntegrationTests.GraphQL/GraphQLTests.cs

```diff
@@ -219,7 +220,10 @@ private HttpStatusCode SubmitRequest(int aspNetCorePort, RequestInfo requestInfo
     {
         try
         {
+            #pragma warning disable SYSLIB0014 // suppress error SYSLIB0014: 'WebRequest.Create(string)' is obsolete: 'WebRequest, HttpWebRequest, ServicePoint, and WebClient are obsolete. Use HttpClient instead
```

Comment:
Is it possible to convert to HTTP Client?
If so, we could create separate task for it.

Reply:
I initially tried doing it. Then I thought creating an issue. But in the end I thought that it is probably not worth the effort. If you have other opinion you can of course create an issue :+1:

Reply:
It's probably worth creating an issue for it, but I don't think that it's something we need to change right away.

Reply:
The main reason I'm recommending an issue for it, is that I suspect that the old API is going to be removed in a future version of .net (I just don't know which future version).

Reply:
#384